pytorch_lightning/callbacks/quantization.py

```diff
@@ -318,3 +326,82 @@ def on_predict_start(self, trainer: "pl.Trainer", pl_module: "pl.LightningModule
     def on_predict_end(self, trainer: "pl.Trainer", pl_module: "pl.LightningModule") -> None:
         if "predict" in self._observer_disabled_stages:
             self._restore_last_observer_enabled()
+
+
+class INCQuantization(Callback):
```

Comment:
Let's add some tests for this component.

Reply:
Done